src/main/utility/mod.rs

```diff
@@ -394,7 +396,8 @@ mod export {
             **ABORTING**"
         );
 
-        if !nix::unistd::isatty(libc::STDOUT_FILENO).unwrap_or(true) {
+        // TODO: Why do we only print this if stdout isn't a terminal?
+        if !std::io::stdout().lock().is_terminal() {
             println!("{error_msg}");
         }
```

Comment:
I think this was intended to be the same reason as what we're doing with error log messages. We always do `eprintln!("{error_msg}");` below, so there's no reason to also do a `println!("{error_msg}");` if they're both being written to a terminal.

Reply:
ah right, I missed the unconditional `eprintln` below